models/posts.rb

```diff
@@ -12,7 +16,7 @@ def initialize(params)
 
 	# Get all posts of the requested student.
 	def get_requested_posts_by_name
-		posts = DATABASE.all_by("name", @student)
+		posts = $database.all_by("name", @student)
```

Comment:
I'm actually not sure about this replacement now, because DATABASE.all_by is supposed to be calling an instance method, whereas the replacement of is supposed to be replacing instances of DATABASE.new